prow/images/whitesource-scanner/wss-unified-agent.config

```diff
@@ -0,0 +1,254 @@
+####################################################################
+# WhiteSource Unified-Agent configuration file
+####################################################################
+##########################################
+# GENERAL SCAN MODE: Files and Package Managers
+##########################################
+  
+checkPolicies=false
+forceCheckAllDependencies=false
+forceUpdate=false
+forceUpdate.failBuildOnPolicyViolation=false
+offline=false
+#ignoreSourceFiles=true
+#scanComment=
+#updateInventory=false
+#resolveAllDependencies=false
+#failErrorLevel=ALL
+#requireKnownSha1=false
+#generateProjectDetailsJson=true
+#generateScanReport=true
+#scanReportTimeoutMinutes=10
+#scanReportFilenameFormat=
+#excludeDependenciesFromNodes=.*commons-io.*,.*maven-model
+
+#projectPerFolder=true
+#projectPerFolderIncludes=
+#projectPerFolderExcludes=
+
+#analyzeFrameworks=true
+#analyzeFrameworksReference=
+
+#wss.connectionTimeoutMinutes=60
+# Change the below URL to your WhiteSource server.
+# Use the 'WhiteSource Server URL' which can be retrieved
+# from your 'Profile' page on the 'Server URLs' panel.
+# Then, add the '/agent' path to it.
+wss.url=https://saas.whitesourcesoftware.com/agent
+
+#npm.resolveDependencies=false
+#npm.ignoreSourceFiles=false
+#npm.includeDevDependencies=true
+#npm.runPreStep=true
+#npm.ignoreNpmLsErrors=true
+#npm.ignoreScripts=true
+#npm.yarnProject=true
+#npm.accessToken=
+#npm.identifyByNameAndVersion=true
+#npm.yarn.frozenLockfile=true
+#npm.resolveMainPackageJsonOnly=true
+#npm.projectNameFromDependencyFile=true
+
+#bower.resolveDependencies=false
```

Comment:
Do we really need all those commented lines?

Reply:
This is just the standard config, in case we want to enable more scanning (e.g. different languages) it is easy to re-use.